### PR TITLE
added support for colours

### LINKED ISSUE #199: Add logging to BackEndApp
## Feature Request

Is your feature request related to an existing [Issue](https://www.github.com/Auto-DL/Generator/issues) or a new implementation?

No

### Feature Description

Add logging support to BackEndApp to make it easier to read the logs and debug errors. 

### Possible Solution

Use native Python3 logging library or [coloredlogs](https://pypi.org/project/coloredlogs/) for visually pleasing logs

### Additional context

I usually use the following configuration:
```python
bold_seq = '\033[1m'
colorlog_format = (
    f'{bold_seq}'
    '%(log_color)s'
    '%(asctime)s | %(name)s/%(funcName)s | '
    '%(levelname)s:%(reset)s %(message)s'
)
colorlog.basicConfig(format=colorlog_format,
                     level=logging.DEBUG, datefmt='%d/%m/%Y %H:%M:%S')
```

### PR DESCRIPTION
**Pull request by shubhjhawar**
# Pull Request

## What does this PR do?
Fixes #199 

## What part does this affect?
- [ ] FrontEnd.
- [x] BackEnd.
- [ ] Documentation.
- [ ] Other. (Please specify below)

## Before submitting
- [x] Was this discussed/approved via a GitHub issue or slack?
- [x] Did you read the [contributor guideline](https://github.com/Auto-DL/Auto-DL/blob/v1-beta/CONTRIBUTING.md)?
- [x] Did you ensure that there aren't any other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change?
- [x] Did you make sure the title is self-explanatory and the description concisely explains the PR?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure the code is clean and docstrings have been added or updated as required?
- [x] Did you make sure the code is linted/formatted locally prior to submission? (using `black` and/or `prettier`)
- [x] Did you make sure to update the documentation with your changes? (if necessary)

## PR review
Anyone in the community is free to review the PR once the tests have passed.

Thank you for contributing to [AutoDL](https://github.com/Auto-DL/Auto-DL). We look forward to your continued support.

### CODE REVIEW

File: BackEndApp/BackEndApp/settings.py [hunk 1]
```diff
@@ -125,3 +126,38 @@
...
 EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
+
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'colored': {
+			'()': 'colorlog.ColoredFormatter',
+			'format': "%(log_color)s %(asctime)s %(name)-12s %(levelname)-8s %(message)s"
+        },
```
Comment (RusherRG) [inline BackEndApp/BackEndApp/settings.py:137]: Instead of having a separate formatter `colored` just add it to console so it's the default

**Review by RusherRG — CHANGES_REQUESTED** [review]

File: BackEndApp/BackEndApp/settings.py (hunk 1 above)
Comment (shubhjhawar) [reply BackEndApp/BackEndApp/settings.py:137]: done, please have a look.

Comment (RusherRG) [reply BackEndApp/BackEndApp/settings.py:137]: Nice :v:

**Review by RusherRG — APPROVED** [review]

**Comment by ADI10HERO** [comment]
Error 1: 
image

Error 2:
image


My suggestion:
Add appropriate steps in `README.md` and/or update `requirements.txt` (probably under #dev comment)

**Comment by ADI10HERO** [comment]
@RusherRG :(

**Pull request closed without merging**